siliconcompiler/tools/klayout/klayout_show.py

```diff
@@ -90,9 +90,8 @@ def __screenshot(schema, layout_view, output_path):
                                          'show_vertical_resolution',
                                          step=step, index=index)[0])
 
-    gds_img = layout_view.get_image(horizontal_resolution, vertical_resolution)
     print(f'[INFO] Saving screenshot to {output_path}')
-    gds_img.save(output_path, 'PNG')
+    layout_view.save_image(output_path, horizontal_resolution, vertical_resolution)
```

Comment:
At least the klayout version from litex-hub seems to be built without Qt bindings, which makes `LayoutView::get_image()` unavailable. This function returns a `QtImage` of the layout which can then be saved. There seems to be no other use of the Qt bindings except for saving a screenshot of the layout, which can now be handled via `LayoutView::save_image()` instead.

Documentation for the two functions:
https://www.klayout.de/doc-qt5/code/class_LayoutView.html#method100
https://www.klayout.de/doc-qt5/code/class_LayoutView.html#method203

Reply:
we have a few other tasks (screenshot) which would be impacted by this.

Reply:
They should all work with these changes because no task actually uses the Qt object for anything other than saving to a file, which can be done through `save_image()` instead without the need for going through Qt on the Python side.
I'll branch this change off to its own PR and test it some more. Then this PR can focus on the conda/CI changes.